src/rust/engine/dep_inference/src/python/tests.rs

```diff
@@ -558,3 +558,4 @@ fn syntax_errors_and_other_fun() {
   assert_imports("try:...\nexcept (x=1):import a", &["a"]);
   assert_imports("foo()", &[]);
 }
+//comment
```

Comment:
I suspect this is an "oopsie" to trigger new hash?

Reply:
or... it isn't ...... yet....